processor/stream/test_approved_stream_result/testIntegrationResultReadError.approved.json

```diff
@@ -1,5 +1,5 @@
 {
-    "accepted": 4,
+    "accepted": 3,
```

Comment:
why did this one change?

Reply:
I removed the newline after the last event in [transactions.ndjson](https://github.com/elastic/apm-server/pull/4261/files/a45f7abe6d411b1e80b3568e3f35ed5c72bb4514#diff-11da2c3d2bc8519471aede56bbcbe654), to ensure the changes in the stream processor still work as expected when encountering an `io.EOF`. Removing the newline in `master` branch leads to the exact same result.

Reply:
OK, thanks for the explanation